xgi/classes/reportviews.py

```diff
@@ -394,6 +394,14 @@ def members(self, e):
             Returns an error if the user tries passing in a slice or if
             the edge ID does not exist in the hypergraph.
         """
+        if e is None:
+            if dtype is dict:
+                return self._ids.copy()
+            elif dtype is list:
+                return [v for _, v in self._ids.items()]
```

Comment:
What about making this `list(self._ids.values())`?

Reply:
Done